vertx-mutiny-clients/vertx-mutiny-core/src/test/java/io/vertx/mutiny/test/CoreTest.java

```diff
@@ -90,7 +90,8 @@ public void testWebSocket() {
             ws.toMulti()
                     .subscribe().with(msg -> {
                         serverReceived.incrementAndGet();
-                        ws.writeTextMessage("pong");
+                        ws.writeTextMessage("pong").subscribe().with(v -> {
```

Comment:
writeTextAndForget would have been enough.

Reply:
ah I did not spot it